src/pages/home-page/ui/component.tsx

```diff
@@ -0,0 +1,47 @@
+import { useStore } from 'effector-react';
+import React from 'react';
+import { Button, Container, Paper, TextField } from 'shared/ui';
+import {
+  $passwordStore,
+  $usernameStore,
+  handleSubmit as onSubmit,
+  setPassword,
+  setUsername,
+} from '../model';
```

Comment:
here we are not exporting directly as we are using a public API approach we are exporting the whole model from the model directory with the needed methods

so in model, you will have index.ts with `export * from './store'`

and on the UI you will just `import * as model from '../model'`